pkg/keyspace/tso_keyspace_group.go

```diff
@@ -146,6 +148,47 @@ func (m *GroupManager) Close() {
 	m.wg.Wait()
 }
 
+func (m *GroupManager) allocDefaultNodesForKeyspaceGroup() {
```

Comment:
```suggestion
func (m *GroupManager) allocNodesToAllKeyspaceGroups() {
```

It seems that here we try to alloc nodes to all keyspace groups, so the original name is confusing.